[imaging_browser] Updated test plan (#6472)
aces/Loris: modules/imaging_browser/test/TestPlan.md

25. On the view session page, click the button "Visit Level Feedback".
26. The entries in this dialog should be editable when a user has the permission `imaging_browser_qc`.
27. Try editing comments (adding new ones, deleting old ones). Click Save and ensure the data is saved.

### Test the Candidate Dashboard widget

> 28. Go to the candidate dashboard in the candidate module and check the Imaging QC Summary.

Reviewer: ```suggestion
28. Go to the candidate dashboard in the candidate module and check the Imaging QC Summary widget.
```

> 28. Go to the candidate dashboard in the candidate module and check the Imaging QC Summary widget.

Reviewer 2: It seems to be missing tests for the user functionality that is described in the widget itself (ie hovering, clicking..)

>     - For each visit, check that the QC status displayed matches the Imaging Browser Module.

Reviewer 2: This is missing tests for the breakdowns by status

>     - For each visit, check that the QC status corresponds to the one in the Imaging Browser Module.

Reviewer: ```suggestion
    - For each visit, check that the QC status displayed matches the Imaging Browser Module.
```

>     - Check that all the files from the Imaging Browser Module appears in the widget for all the visits.

Reviewer: ```suggestion
    - For a few candidate/visits, check that all the files from the Imaging Browser Module appear in the widget.
```
